packages/contracts-por/package.json

```diff
@@ -5,13 +5,20 @@
   "private": true,
   "scripts": {
     "clean": "rm -rf ./build && hardhat clean",
+    "lint": "yarn lint:sol && yarn lint:ts",
+    "typecheck": "tsc --noEmit",
+    "lint:sol": "solhint 'contracts/**/*.sol' && prettylint 'contracts/**/*.sol'",
```

Comment:
`solhint`, `prettylint` and `prettier` tools are used but not listed on the list of `devDependencies`. They only work because they are installed by the parent `package.json` file. If someone removes or changes the version there, `contracts-por` package breaks. Always declare all used dependencies to avoid such problems.